pt-BR/active_record_querying.md

```diff
@@ -1757,14 +1759,17 @@ Client.find_by_sql("SELECT * FROM clients
 # ]
 ```
 
-`find_by_sql` provides you with a simple way of making custom calls to the database and retrieving instantiated objects.
+`find_by_sql` fornece uma maneira simples de fazer consultas customizadas no banco
+de dados, retornando objetos instanciados.
 
 ### `select_all`
 
-`find_by_sql` has a close relative called `connection#select_all`. `select_all` will retrieve
-objects from the database using custom SQL just like `find_by_sql` but will not instantiate them.
-This method will return an instance of `ActiveRecord::Result` class and calling `to_a` on this
-object would return you an array of hashes where each hash indicates a record.
+O `find_by_sql` possui um parente próximo chamado `connection#select_all`.
+O `select_all` retornará objetos do banco de dados usando SQL customizado,
+assim como o `find_by_sql` mas não os instanciará.
+
+Este método retornará uma instância da classe `ActiveRecord::Result` e chamando
+`to_a` nesse objeto retornaria uma lista de hashes em que cada hash indica um registro.
```

Comment:
Estamos tentando manter todos termos em inglês (como hash, model, view etc) em itálico. Criamos um guia de tradução para tentar documentar tudo: [Manual de Tradução](https://github.com/campuscode/rails-guides-pt-BR/blob/master/TRANSLATION_MANUAL.md#glossário)